internal/core/metadata/rest_addressable.go

```diff
@@ -19,30 +19,37 @@ import (
 	"net/url"
 	"strconv"
 
+	"github.com/edgexfoundry/edgex-go/internal/core/metadata/operators/addressable"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/errorConcept"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 	"github.com/edgexfoundry/go-mod-core-contracts/models"
 	"github.com/gorilla/mux"
-
-	types "github.com/edgexfoundry/edgex-go/internal/core/metadata/errors"
-	"github.com/edgexfoundry/edgex-go/internal/core/metadata/operators/addressable"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
 )
 
+// Global variables
+var addressableErrorConcept errorConcept.AddressableErrorConcept
+
 func restGetAllAddressables(w http.ResponseWriter, _ *http.Request) {
 	op := addressable.NewAddressableLoadAll(Configuration.Service, dbClient, LoggingClient)
 	addressables, err := op.Execute()
 	if err != nil {
-		switch err.(type) {
-		case types.ErrLimitExceeded:
-			http.Error(w, err.Error(), http.StatusRequestEntityTooLarge)
-		default:
-			http.Error(w, err.Error(), http.StatusInternalServerError)
-		}
+		httpErrorHandler.Handle(
+			w,
+			err,
+			[]errorConcept.ErrorConceptType{
+				commonErrorConcept.RequestEntityTooLarge,
+			},
+			defaultErrorConcept.InternalServerError)
 		return
 	}
-	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set(clients.ContentType, clients.ContentTypeJSON)
 	err = json.NewEncoder(w).Encode(&addressables)
 	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+		httpErrorHandler.Handle(
```

Comment:
With addition of alternate Handle() method that does not include an allowable parameters suggested in a separate comment, the calls that are converted to use the alternate method will likely fit on a single line.